src/service/poc_mobile.proto

```diff
@@ -664,3 +667,24 @@ verified_service_provider_boosted_rewards_banned_radio_ingest_report_v1 {
   // epoch
   uint64 timestamp = 3;
 }
+
+message verified_subscriber_mapping_event_req_v1 {
+  /// The id of the discovery mapping enabled subscriber
+  bytes subscriber_id = 1;
+  /// The accumulated mapping points the subscriber has earned this epoch
+  uint64 total_reward_points = 2;
+  /// Unix timestamp in seconds of when the mapping event was generated
+  uint64 timestamp = 3;
+  /// Pubkey of verification mapping event service
+  bytes verification_mapping_pubkey = 4;
```

Comment:
this should be changed back to carrier_mapping_key in order to keep with existing conventions.  The key naming is based on the type of key rather than the name of the service and in this case the key is of type carrier